Products/ZenModel/FileSystem.py

```diff
@@ -110,11 +110,23 @@ class FileSystem(OSComponent):
         )
 
 
+    def _getTotalBlocksAsSum(self):
+        availBlocks = self.cacheRRDValue('availBlocks', None)
+        usedBlocks = self.usedBlocks()
+        if availBlocks and usedBlocks:
```

Comment:
This will evaluate to False if either `availBlocks` or `usedBlocks` has the value zero.  Is this what you intended? Or did you mean to say that both values _have to exist_ which means you should be checking for `None` explicitly.